v2.0/alter-user.md

```diff
@@ -0,0 +1,56 @@
+---
+title: ALTER USER
+summary: The ALTER USER statement can be used to add or change a user's password.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> The `ALTER USER` [statement](sql-statements.html) can be used to add or change a [user's](create-and-manage-users.html) password.
+
+{{site.data.alerts.callout_success}}You can also use the <a href="create-and-manage-users.html#update-a-users-password"><code>cockroach user</code></a> command to add or change a user's password.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Considerations
+
+- Password creation and alteration is supported only in secure clusters for non-`root` users. The `root` user must authenticate with a client certificate and key.
+
+## Required Privileges
+
+The user must have the `INSERT` and `UPDATE` [privileges](privileges.html) on the `system.users` table.
+
+## Synopsis
+
+<section>{% include sql/{{ page.version.version }}/diagrams/alter_user_password.html %}</section>
```

Comment:
Use `<div>` instead of `<section>` - the latter has semantic value which is counter-indicated here, whereas the former does not.